src/test/java/org/broadinstitute/hellbender/engine/FeatureSupportIntegrationTest.java

```diff
@@ -83,6 +92,25 @@ public void testFeaturesAsIntervalsUnrecognizedFormatFile() throws IOException {
         testSpec.executeTest("testFeaturesAsIntervals", this);
     }
 
+    @Test
+    public void testSitesOnlyMode() {
+        File out = createTempFile("GTStrippedOutput", "vcf");
+        String[] args = new String[] {
+                "-V",  FEATURE_INTEGRATION_TEST_DIRECTORY + "vcf_with_genotypes.vcf",
+                "--" + StandardArgumentDefinitions.SITES_ONLY_LONG_NAME,
```

Comment:
Can you confirm that this test fails if you remove the `--sites-only` arg temporarily?

Reply:
just checked, it fails without --sites-only